### DISCUSSION TITLE
Multiple systems in dirWatch with SDRTrunk?

### DISCUSSION THREAD

**Discussion by canuckcam** [chuot/rdio-scanner#247] [Q&A] [answered]
Is there a way to separate systems with the dirWatch method using SDRTrunk? I have an AirSpy receiving two trunking systems and would like to feed both to Rdio-Scanner. I can't seem to configure/let dirWatch know how to separate the systems as they're labelled in the filenames. Is this even possible?

Thanks!

**Comment by GitGoorilla** [comment, marked as answer]
Are you referring to having 2 systems in 1 rdio-scanner instance?

In rdio-scanner, you would create a "New System". The "Label" for the system needs to match exactly what your "System" label is in SDRTrunk. You can choose to auto-populate your Talkgroups or enter them manually. 

Example:
![Screenshot 2022-08-07 231442](https://user-images.githubusercontent.com/88252409/183337898-fdd00b6a-53fb-46b5-92f0-79e38510b4fe.png)


![Screenshot 2022-08-07 231641](https://user-images.githubusercontent.com/88252409/183338079-52796315-e883-448e-8002-83e6e7cf9667.png)

**Comment by chuot** [comment]
Thanks @GitGoorilla for answering this one.

**Comment by canuckcam** [comment]
Thank you - somewhere, something went wrong but I'm not able to recreate it. I had the exact system name but the DirWatch kept failing. In SDRTrunk, a new system I made worked fine. (!!?) So I deleted the site name, saved, re-entered the same Site name, and now it works! (!!?) 

![Screen Shot 2022-08-10 at 12 09 20 AM](https://user-images.githubusercontent.com/30516436/183812679-98970b1a-a43f-417f-89e6-5e5823953f96.png)
